rotkehlchen/tests/unit/globaldb/test_globaldb_cache.py

```diff
@@ -252,13 +254,6 @@ def test_velodrome_cache(optimism_inquirer):
     assert all(entry in addressbook_entries for entry in VELODROME_SOME_EXPECTED_ADDRESBOOK_ENTRIES)  # noqa: E501
     assert all((identifier,) in asset_identifiers for identifier in VELODROME_SOME_EXPECTED_ASSETS)
 
-    # New messages are sent only after a delay of 5 secs and the test runs fast
-    # due to the vcr, so only the first message from each loop gets sent.
-    assert mock_notify.call_args_list == [
-        make_call_object(CPT_VELODROME, ChainID.OPTIMISM, 0, 200),
-        make_call_object(CPT_VELODROME, ChainID.OPTIMISM, 1, 282),
-    ]
```

Comment:
test passes locally with 100 but ci gets 105 due to cassette processing speed differences, so removed the check